server/internal/store/org.go

```diff
@@ -0,0 +1,13 @@
+package store
+
+import "gorm.io/gorm"
+
+// Organization is a model for organization
+type Organization struct {
+	gorm.Model
+
+	TenantID       string `gorm:"uniqueIndex:tenant_id_org_id"`
+	OrganizationID string `gorm:"uniqueIndex:tenant_id_org_id"`
```

Comment:
It might be better to just have a unique constraint on `OrganizationID` so that the ID is globally unique. This might not be applicable here, but having a globally unique ID makes the implementation simple in general (e.g., when we query an org with an internal gRPC where tenant ID is not available).

Here is an example change I made to Model Manger: https://github.com/llm-operator/model-manager/commit/0141f70fb0a5ffc74c9e9cc54e830fd3947ae0d3#diff-1c71dfdcfa9a9f99f62f7c29af00ebee1842982b5b42af7c111a262d87a1e204

We can also change later.